src/scripts/get-wpt-revision.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+# Copyright 2018 The WPT Dashboard Project. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import argparse
+import contextlib
+import httplib
+import json
+import urlparse
+
+
+@contextlib.contextmanager
+def request(method, url):
```

Comment:
This seems a little complicated because of the low-level nature of `httplib`. Is `requests` not available here?

Reply:
`requests` is not available here because I've been trying to limit external dependencies to Buildbot itself. That helps to make it possible to "drop in" these scripts, and it limits complexity in the project configuration (e.g. "install this `requirements.txt` when this script is present, install that `requirements.txt` when that script is present..."). The `request` function is called once and could easily be inlined; the separation is my attempt to encapsulate those low-level details so that the simplicity of the business logic is more obvious.